features/logging_in_to_the_application.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Login to the app
+
+as a user
+in order to use the application
+I would like to login to my account
+
+
+Background:
+Given the following user exists
```

Comment:
`user` should be `users` this is a step definition that allows you to create many users and that should be reflected in the step definition message/title.